packages/ckeditor5-alignment/tests/alignmentediting.js

```diff
@@ -148,6 +176,30 @@ describe( 'AlignmentEditing', () => {
 
 			expect( editor.getData() ).to.equal( '<p>x</p>' );
 		} );
+
+		it( 'uses class when alignment.classNames is set', async () => {
+			return VirtualTestEditor
+				.create( {
+					plugins: [ AlignmentEditing, Paragraph ],
+					alignment: {
+						classNames: [ 'foo-left', 'foo-right', 'foo-center', 'foo-justify' ]
+					}
+				} )
+				.then( newEditor => {
+					const model = newEditor.model;
+
+					setModelData( model, '<paragraph alignment="center">[]x</paragraph>' );
+
+					expect( newEditor.getData() ).to.equal( '<p class="foo-center">x</p>' );
+
+					newEditor.execute( 'alignment', { value: 'left' } );
+
+					expect( getModelData( model ) ).to.equal( '<paragraph>[]x</paragraph>' );
```

Comment:
While `text-align:left` is redundant in LTR the same way as `text-align:right` is redundant in RTL, I wonder if, when using classes, we should **always** assign them to DOM nodes.

WDYT, @Reinmar ?

Reply:
We decided to leave the same behavior as with inline style.